.github/workflows/pypi.yml

```diff
@@ -23,11 +23,10 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.x
+          python-version: 3.9
       - name: Build
         shell: bash -l {0}
         run: |
-          pip install setuptools==58.5.3
```

Comment:
@pstradio I would do both, set python to 3.9 and keep "pip install -U setuptools" to be sure